docs/architecture/self-hosted/resource-monitoring.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Resource monitoring
+id: resource-monitoring
+description: Find an overview of resource monitoring, alert configuration, and scaling recommendations
+sidebar_position: 5
+tags:
+  - scaling
+  - resources
+  - monitoring
+---
+
+This page provides an overview of resource monitoring, alert configuration, and scaling recommendations for effectively managing Prophecy deployments.
+
+## Monitoring
+
+To have a better understanding of how Prophecy is using resources, you can find an overview in **Settings > Admin > Monitoring**.
+
+![Monitoring settings](./img/monitoring.png)
+
+## Alerts
+
+To set up automated alerts about your resource usage, visit the page on [Alerts Configuration](docs/architecture/self-hosted/configurations/configure-alerts.md).
+
+## Scaling
+
+Scaling is taken care of by the **Autoscaler** component in our platform. Therefore, if something happens like the number of
+users increases by 10 times, the Prophecy deployment can be scaled appropriately.
+
+The following are estimated recommended cluster sizes depending on the number of concurrent users:
+
+| Number of users           | 25       | 50       | 150       |
```

Comment:
@prashanth26 This was a copy-paste from previous docs. Is the information/are the numbers still accurate?